site/content/en/docs/handbook/config.md

```diff
@@ -114,14 +114,12 @@ Some features can only be accessed by minikube specific environment variables, h
 
 * **MINIKUBE_IN_STYLE** - (bool) manually sets whether or not emoji and colors should appear in minikube. Set to false or 0 to disable this feature, true or 1 to force it to be turned on.
 
-* **MINIKUBE_WANTUPDATENOTIFICATION** - (bool) sets whether the user wants an update notification for new minikube versions
```

Comment:
are these being delete automatically? or do we not use them anymore?

Reply:
I manually deleted them, I searched for the string and it wasn't included anywhere in the project. They exist in the config now, so I think they got moved there but this reference wasn't removed.